docker-compose.yml

```diff
@@ -30,7 +30,6 @@ x-common-variables:
     NETWORK_CONNECTION_HISTORY_TABLE: ${DEPLOYMENT_NAME}-network_connection_history_table
     OUTBOUND_CONNECTION_HISTORY_TABLE: ${DEPLOYMENT_NAME}-outbound_connection_history_table
     PROCESS_HISTORY_TABLE: ${DEPLOYMENT_NAME}-process_history_table
```

Comment:
We should be able to remove all of the tables except for the static/session ones.

Reply:
Awesome; I should've dug into this more but glad to hear we can delete these.